src/agent/src/unix_daemon.hpp

```diff
@@ -13,7 +13,7 @@ namespace unix_daemon
     public:
         /// @brief Default constructor
         /// @details Creates a lock file when the instance is created
-        LockFileHandler();
+        LockFileHandler(std::string lockFilePath);
```

Comment:
Add parameter to constructor description.

Reply:
Done!